xcode.tera

```diff
@@ -1,33 +1,49 @@
+---
+whiskers:
+  version: 2.4.0
+  matrix:
+    - flavor
+  filename: "themes/Catppuccin {{ flavor.name }}.xccolortheme"
+---
+
+{%- macro val(val) -%}
+{{ val / 255 | trunc(places=6) | float * 1000000 | round / 1000000 }}
```

Comment:
For context, the multiplication by 1000000, rounding, and division by 1000000 is to round numbers to 6 places if needed but simplify them as much as possible otherwise. As you can see by the diff, this exactly matches how it was before.